docs/website/docs/examples/qdrant_zendesk/index.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Similarity Searching with Qdrant
+description: Learn how to use the dlt source, Zendesk and dlt destination, Qdrant to conduct a similarity search on your tickets data.
+keywords: [similarity search, example]
+---
+import Header from '../_examples-qdrant-header.md';
+
+<Header
+    intro="In this tutorial, you will learn how to do use dlt to store your
+    vectorized Zendesk tickets data in the dlt destination; Qdrant. You can
+    use Qdrant's vectorization and similarity searching capabilities on your tickets data,
+    while using dlt as a medium to automate your pipeline."
+    slug="qdrant_zendesk"
+    run_file="qdrant-snippets"
+/>
+
+This article outlines a system to map vectorized ticket data from Zendesk to Qdrant, similar to our guide on the topic concerning [Weaviate](https://dlthub.com/docs/dlt-ecosystem/destinations/qdrant). In this example, we will:
+- Connect to our Zendesk source.
+- Extract tickets data from our Zendesk source.
+- Create a dlt pipeline with Qdrant as destination.
+- Vectorize/embed the tickets data from Zendesk.
+- Pass the vectorized data to be stored in Qdrant via the dlt pipeline.
+- Query data that we stored in Qdrant.
+- Explore the similarity search results.
+
+First, configure the destination credentials for [Qdrant](https://dlthub.com/docs/dlt-ecosystem/destinations/qdrant#setup-guide) and [Zendesk](https://dlthub.com/docs/walkthroughs/zendesk-weaviate#configuration) in .dlt/secrets.toml.
+
+## Connect to Zendesk and load tickets data
+
+<!--@@@DLT_SNIPPET_START ./code/qdrant-snippets.py::zendesk_conn-->
+```py
+from typing import Optional, Dict, Any, Tuple
+
+import dlt
+from dlt.common import pendulum
+from dlt.common.time import ensure_pendulum_datetime
+from dlt.common.typing import TAnyDateTime
+from dlt.sources.helpers.requests import client
+from dlt.destinations.qdrant import qdrant_adapter
+from qdrant_client import QdrantClient
+
+from dlt.common.configuration.inject import with_config
+
+# helper function to fix the datetime format
+def _parse_date_or_none(value: Optional[str]) -> Optional[pendulum.DateTime]:
```

Comment:
you can hide this functions (_parse_date_or_none and _fix_date)  as well, just move them with `get_pages` out of ` # @@@DLT_SNIPPET_START zendesk_conn` tag